### ISSUE TITLE
How to use this es-change-feed-plugin after installation?

### ISSUE THREAD

**Issue by liyuTony35** [ForgeRock/es-change-feed-plugin#6]
After the successful installation of this plugin v2.4.0 , I have no idea how to interact with this Plugin to 
get recent data-changes.  Can you give me an example of using this plugin?
the ways I have tried were:
**curl -XGET  http://localhost:9400/ws/_changes,**
but receive a prompt of  no resource available.

ps: Is this plugin now really a  complete artifact? @jurgc11

**Comment by zoltantarcsay** [comment]
This is a websocket, so you can't talk to it over HTTP. Try a websocket client.

**Comment by rafaelfamorim** [comment]
I've installed v5.4.0 in a 6.0.0 ES instance, following these instructions: https://github.com/ForgeRock/es-change-feed-plugin/issues/4 
Instalation was successful, but after restart ES, I don't know how to make sure if it's working. I've tried to connect to websocket but it didn't work, it seems that websocket hasn't started. Is there any way to check plugin status? To know if it's running or not?
